packages/ui/src/types/authenticator/attributes.ts

```diff
@@ -61,3 +61,9 @@ export const authFieldsWithDefaults = [
 
 /** Input fields that we provide default fields with */
 export type AuthFieldsWithDefaults = typeof authFieldsWithDefaults[number];
+
+export const isAuthFieldsWithDefaults = (
+  field: string
+): field is AuthFieldsWithDefaults => {
+  return authFieldsWithDefaults.includes(field as AuthFieldsWithDefaults);
```

Comment:
Curious if this type assertion required

Reply:
It is, since `AuthFieldsWithDefaults` is an array of strongly typed string unions. Without it, we'll get these errors:

```
Argument of type 'string' is not assignable to parameter of type '"birthdate" | "email" | "family_name" | "given_name" | "middle_name" | "name" | "nickname" | "phone_number" | "preferred_username" | "profile" | "website" | "username" | "password" | "confirm_password" | "confirmation_code"'.ts(2345)
```